apps-rendering/src/components/LiveBlocks/index.tsx

```diff
@@ -61,34 +118,14 @@ const LiveBlocks: FC<LiveBlocksProps> = ({ blocks, format, pageNumber }) => {
 						map((pinned) => block.id === pinned.id),
 						withDefault<boolean>(false),
 					)}
-					supportsDarkMode={true}
-					contributors={block.contributors.map(
-						contributorToBlockContributor,
-					)}
-				>
-					{renderAll(format, partition(block.body).oks)}
-
-					<footer
-						css={css`
-							display: flex;
-							justify-content: end;
-						`}
-					>
-						{block.lastModified > block.firstPublished && (
-							<LastUpdated
-								lastUpdated={block.lastModified}
-								lastUpdatedDisplay={formatUTCTimeDateTz(
-									block.lastModified,
-								)}
-							/>
-						)}
-					</footer>
-				</LiveBlockContainer>
+				></LiveBlock>
```

Comment:
You could use a self-closing tag for this:

```suggestion
				/>
```